dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/programrule/SetMandatoryFieldValidator.java

```diff
@@ -28,20 +28,31 @@
 
 package org.hisp.dhis.tracker.programrule;
 
```

Comment:
I see this in many places in NTI actually. Package declaration is usually before the Copyright.